lib/uffizzi/cli/project.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'io/console'
+require 'uffizzi'
+require 'uffizzi/auth_helper'
+require 'thor'
+
+module Uffizzi
+  class CLI::Project < Thor
+    desc 'compose', 'compose'
+    method_option :file, required: false, aliases: '-f'
+    require_relative 'project/compose'
+    subcommand 'compose', Uffizzi::CLI::Project::Compose
+
+    desc 'list', 'list'
+    def list
+      Project.new(options, 'list').run
+    end
+
+    class Project
+      include ApiClient
+
+      def initialize(options, command)
+        @options = options
+        @command = command
+      end
+
+      def run
+        return unless Uffizzi::AuthHelper.signed_in?
+
+        case @command
+        when 'list'
+          handle_list_command
+        end
+      end
+
+      private
+
+      def handle_list_command
+        hostname = ConfigFile.read_option(:hostname)
+        response = fetch_projects(hostname)
+
+        if response[:code] == Net::HTTPOK
+          handle_succeed_response(response)
+        else
+          handle_failed_response(response)
+        end
+      end
+
+      def handle_failed_response(response)
+        print_errors(response[:body][:errors])
+      end
+
+      def handle_succeed_response(response)
+        projects = response[:body][:projects]
+        if projects.empty?
+          Uffizzi.ui.say('No projects related to this email')
+          return
+        end
+        if projects.size == 1
```

Comment:
What is this for?

Reply:
we automatically set project slug in config file if there is just one project

Reply:
`one?` method doesn't work that way, so there's still must be `projects.size == 1` condition